src/ansys/units/quantity.py

```diff
@@ -176,41 +158,59 @@ def to(self, to_units: Union[ansunits.Unit, str]) -> "Quantity":
 
         return Quantity(value=new_value, units=to_units)
 
+    def __float__(self):
```

Comment:
I am minded towards not exposing this. But we need a discussion about because it will have an impact.

Reply:
If I've read things right then it looks like tests will fail if we don't expose it.   i.e. we have tests passing quantity like this:
```
val = math.sqrt(q)
```
Here's what numpy does:
```
>>> import numpy as np
>>> a = np.array([1,2,3,4])
>>> import math
>>> math.sqrt(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: only length-1 arrays can be converted to Python scalars
```
So, does not walk the array or anything, which is fine, but if a Quantity is a scalar value it seems reasonable to try and implement it no?  Though, maybe we run into issues like what does it mean to do `sqrt(1 [ft])`, is that `1 [ft^0.5]` or do we convert to SI then return the SI value.